src/js/components/Notification/stories/TitleAndMessage.js

```diff
@@ -21,6 +21,7 @@ const TitleAndMessageNotification = () => {
           title="Status Title"
           message="Messages should be at max two lines of text."
           onClose={onClose}
+          time={4000}
```

Comment:
Minor comment but I think we can just have `time={4000}` in the Notification/Title story and leave it out of this story so we should both the default behavior and the behavior of a toast when `time` is defined.
```suggestion
```